krkn/scenario_plugins/node_actions/common_node_functions.py

```diff
@@ -4,12 +4,10 @@
 import logging
 import paramiko
 from krkn_lib.models.k8s import AffectedNode
-import krkn.invoke.command as runcommand
 from krkn_lib.k8s import KrknKubernetes
 from krkn_lib.models.k8s import AffectedNode, AffectedNodeStatus
 from krkn_lib.models.k8s import AffectedNode
-
-node_general = False
+from krkn_lib.models.k8s import KrknKubernetes
```

Comment:
@levi178u Thanks for the PR! Functional tests are failing, can you take a look at it when you get a chance please?
```
ImportError: cannot import name 'KrknKubernetes' from 'krkn_lib.models.k8s' (/opt/hostedtoolcache/Python/3.9.22/x64/lib/python3.9/site-packages/krkn_lib/models/k8s/__init__.py)
```